Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -57,6 +61,10 @@ The steps required to calculate and report an SCI score are:
 
 ## Methodology Summary
 
+### General
```

Comment:
We can use "General" but will discuss with the WG on whether to use "Introduction"